lib/lrama/option_parser.rb

```diff
@@ -91,7 +91,7 @@ def parse_by_option_parser(argv)
       end
     end
 
-    BISON_REPORTS = %w[states itemsets lookaheads solved counterexamples cex all none]
+    BISON_REPORTS = %w[terms states itemsets lookaheads solved counterexamples cex all none]
     OTHER_REPORTS = %w[verbose]
```

Comment:
Could you move to `OTHER_REPORTS`?

It because below:

```
❯ bison -h
Usage: bison [OPTION]... FILE
: (snip)
Output Files:
  -H, --header=[FILE]           also produce a header file
  -d                            likewise but cannot specify FILE (for POSIX Yacc)
  -r, --report=THINGS           also produce details on the automaton
: (snip)
THINGS is a list of comma separated words that can include:
  states            describe the states
  itemsets          complete the core item sets with their closure
  lookaheads        explicitly associate lookahead tokens to items
  solved            describe shift/reduce conflicts solving
  counterexamples, cex
                    generate conflict counterexamples
  all               include all the above information
  none              disable the report
```

```suggestion
    BISON_REPORTS = %w[terms states itemsets lookaheads solved counterexamples cex all none]
    OTHER_REPORTS = %w[verbose]
```

Reply:
OK, it's moved.